package/cfg/rke2-cis-1.5/master.yaml

```diff
@@ -0,0 +1,1081 @@
+        For example,
+        chown root:root /etc/kubernetes/admin.conf
+      scored: true
+      type: skip
+
+    - id: 1.1.15
+      text: "Ensure that the scheduler.conf file permissions are set to 644 or more restrictive (Scored)"
+      audit: "/bin/sh -c 'if test -e /etc/kubernetes/scheduler.conf; then stat -c %a /etc/kubernetes/scheduler.conf; fi'"
+      tests:
+        test_items:
+          - flag: "644"
+            compare:
+              op: eq
+              value: "644"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chmod 644 /etc/kubernetes/scheduler.conf
+      scored: true
+      type: skip
+
+    - id: 1.1.16
+      text: "Ensure that the scheduler.conf file ownership is set to root:root (Scored)"
+      audit: "/bin/sh -c 'if test -e /etc/kubernetes/scheduler.conf; then stat -c %U:%G /etc/kubernetes/scheduler.conf; fi'"
+      tests:
+        test_items:
+          - flag: "root:root"
+            compare:
+              op: eq
+              value: "root:root"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chown root:root /etc/kubernetes/scheduler.conf
+      scored: true
+      type: skip
+
+    - id: 1.1.17
+      text: "Ensure that the controller-manager.conf file permissions are set to 644 or more restrictive (Scored)"
+      audit: "/bin/sh -c 'if test -e /etc/kubernetes/controller-manager.conf; then stat -c %a /etc/kubernetes/controller-manager.conf; fi'"
+      tests:
+        test_items:
+          - flag: "644"
+            compare:
+              op: eq
+              value: "644"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chmod 644 /etc/kubernetes/controller-manager.conf
+      scored: true
+      type: skip
+
+    - id: 1.1.18
+      text: "Ensure that the controller-manager.conf file ownership is set to root:root (Scored)"
+      audit: "/bin/sh -c 'if test -e /etc/kubernetes/controller-manager.conf; then stat -c %U:%G /etc/kubernetes/controller-manager.conf; fi'"
+      tests:
+        test_items:
+          - flag: "root:root"
+            compare:
+              op: eq
+              value: "root:root"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chown root:root /etc/kubernetes/controller-manager.conf
+      scored: true
+      type: skip
+
+    - id: 1.1.19
+      text: "Ensure that the Kubernetes PKI directory and file ownership is set to root:root (Scored)"
+      audit: "ls -laR /etc/kubernetes/pki/"
+      type: "manual"
```

Comment:
Should not be manual. rke1 actually has a script for it:
https://github.com/rancher/security-scan/blob/master/package/cfg/rke-cis-1.5-hardened/etcd.yaml#L46